Gemfile.lock

```diff
@@ -188,6 +189,7 @@ GEM
       slop (~> 3.4)
     pry-rails (0.3.4)
       pry (>= 0.9.10)
+    public_suffix (2.0.5)
```

Comment:
Nah, just some tongue and cheek humor from moi. I've been working a lot on making our Gemfile smaller such as removing some rails dependencies we don't use (such as actionmailer) to getting rid of some development dependencies (such as web-console).